scripts/lib/mkcloud-common.sh

```diff
@@ -562,6 +562,10 @@ function common_set_slesversions
             sesversion=5
         ;;
     esac
+
+    if [ $want_external_ceph -gt 0 ]; then
+        sesversion=$want_external_ceph
```

Comment:
This seems like one of the two variables is useless.
Maybe just call it `$want_ses_version`.

Reply:
$sesversion is already used in several places, so if the variable gets renamed, it would have to be updated there too.

Reply:
From this line, it does indeed look that way, but we need a way to indicate we are deploying Ceph, but don't do it internally, however we do need things like the relevant repos on the nodes. We use $want_external_ceph to indicate this. Maybe I'm overloading the variable to also store the sesversion required.

Would it make more sense instead of having 1 variable that both says I want an external ceph and what version it needs to be:

  $want_external_ceph (0|4|5)

Instead break it down into 2 variables:

  $want_external_ceph (0|1)
  $want_ses_version (4|5)

The latter could be used to override the sesversion variable?

Reply:
Maybe something like this (see changed patchset) Noting I'd need to squish the last 2 patches together and update the main commit message to be correct.

But this will add a `want_ses_version` which allows a user to override the sesversion variable used. e.g. export want_ses_version=5

Then the `want_external_ceph` can be (0|1) and only used to tell mkcloud, we don't want to deployceph internally, we'll do that with lonelynode, register and deployexternalceph steps later. But in the meantime we need the correct ses repos available on the crowbar host etc.